### ISSUE TITLE
Documentation for sortedMap is broken

### ISSUE THREAD

**Issue by matklad** [swannodette/mori#172]
The documentation for `sortedMap` is broken: there is no `sortedMap` in the navigation pane. 

It is present in the main text, just above the `set` but it is named `sorted_map`

**Comment by binarykitchen** [comment]
yep, missing and would like to see examples how to turn JS objects to a sorted map directly similar to `toClj()`
